web/client-api/src/main/java/io/deephaven/web/client/api/JsLayoutHints.java

```diff
@@ -48,6 +85,14 @@ public JsLayoutHints parse(String hints) {
             frozenColumns = JsObject.freeze(freezeStr.split(","));
         }
 
+        final String groupsStr = options.get("groups");
+        if (groupsStr != null && !groupsStr.isEmpty()) {
+            List<ColumnGroup> groups =
+                    Arrays.stream(groupsStr.split("\\|")).map(ColumnGroup::new).collect(Collectors.toList());
```

Comment:
probably should freeze each column group instance as they are created, add another .map() in there for it?